backend/fms_core/templates.py

```diff
@@ -120,6 +121,29 @@
       ("Library", "Sample Container Coordinates", "coordinates"),],
 }
 
+LIBRARY_QC_TEMPLATE = {
+  "identity": {"description": "Template to add library QC", "file": static("submission_templates/Library_QC_v3_9_0.xlsx")},
+  "sheets info": [
+      {
+        'name': 'LibraryQC',
+        'headers': ['Library Name', 'Library Container Barcode', 'Library Container Coord', 'Initial Volume (uL)',
+                    'Measured Volume (uL)', 'Volume Used (uL)', 'Strandedness', 'Library size (bp)', 'Concentration (nM)',
+                    'Concentration (ng/uL)', 'NA Quantity (ng)', 'Quality Instrument', 'Quality Flag',
+                    'Quantity Instrument', 'Quantity Flag', 'QC Date (YYYY-MM-DD)', 'Comment'],
+      },
+  ],
+  "prefill info": [
+      # TODO queryset columns
+      ("LibraryQC", "Library Name", "?"),
```

Comment:
This part is a bit magic. Template are based off different models the library and sample prefill assume we'll fill using information from the sample (container move and rename take them from container). This could be parametrized further at some point (we may have to here for the strandedness and library_size).